instagram_app/src/Components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './CommentSection.css';
+
+
+function CommentSection (props) {
+    return(
+<div className="container">
+<div className="userHeader">
+    <img src={props.eachPost.thumbnailUrl} alt="thumbnail" />
+    <h2>{props.eachPost.username}</h2>
+</div>
+<img src={props.eachPost.imageUrl} alt="imageurl" className='postImage'/>
+<div className="likesComments">
+    <img src="https://img.icons8.com/ios/30/000000/like.png"></img>
+    <img src="https://img.icons8.com/ios/30/000000/topic.png"></img>
+</div>
+<div className='counter'>
+    <p className="likes">{props.eachPost.likes} likes</p>
+</div>
+
+
+
```

Comment:
we are also missing the mapping of the comments array to display all the comments for a particular user. I would map over props.eachPost.comments and return some JSX that displays each indiv comment. Let me know if you need help with this.